package.json

```diff
@@ -24,6 +24,7 @@
 		"last 2 Opera versions"
 	],
 	"devDependencies": {
+		"@wordpress/babel-preset-default": "6.2.0",
```

Comment:
I think it's probably now a "prod dependency"

Reply:
Should we remove `@babel/polyfill` from the deps?

Reply:
`@babel/polyfill` - yes we can remove. 

> I think it's probably now a "prod dependency"

It doesn't matter from my perspective. We copy files with webpack so technically they aren't production dependencies, but we can put them in `dependencies` since they are used on production after all :)